gdal/frmts/ecw/frmt_ecw.html

```diff
@@ -0,0 +1,224 @@
+<html>
```

Comment:
Documentation no longer uses HTML files. The doc is now at gdal/doc/source/drivers/raster/ecw.rst